mongodb/config.js

```diff
@@ -0,0 +1,16 @@
+// require('dotenv').load(); //loads .env vars
+var mongoose = require('mongoose');
+
+local: mongoose.connect('mongodb://localhost/test');
+//mongolab:  mongodb://<dbuser>:<dbpassword>@ds035503.mongolab.com:35503/mks-greenfield
```

Comment:
Ah that "local:" was supposed to be in comments.  I uncommented to use local mongoDB.  It is not necessary